types/joint.d.ts

```diff
@@ -532,20 +539,31 @@ export namespace dia {
         endDirections?: ['left' | 'right' | 'top' | 'bottom'];
     }
 
-    interface GridOptions {
+    interface PaperGridOptions {
         color?: string;
         thickness?: number;
         name?: 'dot' | 'fixedDot' | 'mesh' | 'doubleMesh';
         args?: Array<{ [key: string]: any }> | { [key: string]: any };
     }
 
-    interface PaperOptions extends Backbone.ViewOptions<Graph> {
-        el?: string | JQuery | HTMLElement;
-        width?: number | string;
-        height?: number | string;
+    interface PaperBackgroundOptions {
+        color?: string;
+        image?: string;
+        quality?: number;
+        position?: Point | string;
+        size?: Size | string;
+        repeat?: string;
+        opacity?: number;
+        waterMarkAngle?: number;
+    }
+
+    interface PaperOptions extends mvc.ViewOptions<Graph> {
```

Comment:
wouldn't be better to have something like:
```
namespace Paper {
    export interface Options {
        // ...
    }
    export interface GridOptions {
        // ...
    }
    export interface BackgroundOptions {
        // ...
    }
}
```
?

Reply:
Definitely. I'll make changes.